presto-spi/src/main/java/com/facebook/presto/spi/block/BlockEncodingBuffersFactory.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.spi.block;
+
+public class BlockEncodingBuffersFactory
+{
+    private BlockEncodingBuffersFactory()
+    {}
+
+    static BlockEncodingBuffers createBlockEncodingBuffers(Block block)
```

Comment:
Would it make sense to move this method into BlockEncodingSerde ?

Reply:
@mbasmanova What about moving the BlockEncodingBuffersFactory as a static nested class in PartitionedOutputOperator, side by side with PartitionData? PartitionData has a PageSerde, and PageSerge as BlockEncodingSerde, then there are two classes implementing BlockEncodingSerde. Adding createBlockEncodingBuffers() there might complicate the code unnecessarily.

Reply:
@mbasmanova Please excuse my previous response. I moved it to BlockEncodingSerde.